content/case-studies/aptos.md

```diff
@@ -0,0 +1,75 @@
+
+The team configured multi-cloud serverless capabilities such as Google Cloud Functions and Google Cloud Run.
+
+In addition to documenting the migration, the Pulumi team provided import automation and patterns to streamline the transition from Terraform to Pulumi. This approach significantly reduced the need to recreate critical resources and accelerated the import process.
+
+The developers at Aptos Labs reported higher satisfaction with the Pulumi platform than alternative approaches. They also shared that Pulumi helped enforce policies and best practices across their security and cost attribution centers without adding friction.
+
+**Making the switch to Pulumi helped advance Aptos Labs' infrastructure journey in the following ways:**
+
+1. Teams are shipping code faster: Aptos Labs engineers can now prioritize building business value over managing cloud resources using domain-specific code, thanks to Pulumi's streamlined approach.
+2. Developers are enabled in their language of choice: The move to Pulumi allows the developers to use their preferred programming languages, promoting better software engineering practices like encapsulation and DRY (don’t repeat yourself) principles.
+3. Improved development practices and collaboration: Integrating infrastructure management with standard software development techniques enhances code quality and collaboration.
+
+## Removal of Terraform Dependency
+
+With the successful migration from Terraform to Pulumi, Aptos Labs gained access to a more robust and flexible toolset. As a result, Aptos Labs is well-positioned to achieve a more efficient, scalable, and modern infrastructure management approach. Aptos Labs has also allowed engineers to reduce the cognitive burden of understanding a domain-specific language and work in the languages used for critical business functions. With Pulumi, Aptos Labs can now build usable and shareable software abstractions that encapsulate best practices for cloud infrastructure.
+
+## What's Next for Aptos Labs? {#next-steps}
+
+Aptos Labs and Pulumi plan to build on this success to advance improvements in their infrastructure management even further. These efforts will continue to optimize the structure and management of projects at Aptos Labs to achieve greater efficiency, scalability, and consistency.
+
+### Modularization of Projects
+
+Aptos Labs' projects are structured to combine multiple concerns within a single stack. This approach results in managing network, cluster, and application-specific resources together, leading to complexities, redundancies, and excess costs.
```

Comment:
I might reword this; along the lines of "Aptos Labs' projects are structured to combine multiple concerns within a single stack. There are opportunities to optimize and modularize how projects are structured to reduce complexities, redundancies and lower their total cloud spend."